src/search_providers/torrent_csv_search.rs

```diff
@@ -14,6 +14,12 @@ use serde::Deserialize;
 
 use std::error::Error;
 
+#[derive(Debug, Deserialize)]
+pub struct Response {
+    pub torrents: Vec<Entry>,
+    pub next: u64,
```

Comment:
I guess the `next` field is for pagination? I guess it's OK to just return the first page of results.